source/connect/advanced-connection-options/in-use-encryption.txt

```diff
@@ -37,232 +37,230 @@ Procedure
-                 - Required 
-                 - Description 
-
-               * - Tenant Id 
-                 - Yes 
-                 - Identifies the organization for the account.
-
-               * - Client Id 
-                 - Yes 
-                 - Authenticates a registered application.
-
-               * - Client Secret
-                 - Yes
-                 - The client secret to authenticate a registered application.
-
-               * - Identity Platform Endpoint
-                 - Yes
-                 - A host with an optional port. 
-
-               * - Certificate Authority
-                 - No 
-                 - One or more certificate files from trusted Certificate 
-                   Authorities to validate the certificate provided by the deployment.
-               
-               * - Client Certificate and Key
-                 - No 
-                 - Specifies the location of a local .pem file that contains 
-                   either the client's TLS/SSL X.509 certificate or the client's TLS/SSL 
-                   certificate and key.
-
-               * - Client Key Password
-                 - No 
-                 - If the *Client Private Key* is protected with a password, 
-                   you must provide the password.  
+          You can use :v6.0:`Google Cloud Services </core/queryable-encryption/fundamentals/kms-providers/#google-cloud-platform-kms>` to manage your keys.
 
-            .. _kmip-compass-encryption:
+          Specify the following fields:
 
-            KMIP
-            ~~~~
+          .. list-table::
+             :header-rows: 1 
+             :widths: 30 30 70
 
-            You can use :v6.0:`KMIP </core/queryable-encryption/fundamentals/kms-providers/#kmip>` 
-            to manage your keys. 
+             * - Field
+               - Required 
+               - Description 
+
+             * - Service Account Email
+               - Yes 
+               - The service account email to authenticate.
+
+             * - Private Key
+               - Yes 
+               - A base64-encoded private key.
+
+             * - Endpoint
+               - No 
+               - A host with an optional port.
+
+             * - Certificate Authority
+               - No 
+               - One or more certificate files from trusted Certificate 
+                 Authorities to validate the certificate provided by the deployment.
+             
+             * - Client Certificate and Key
+               - No 
+               - Specifies the location of a local .pem file that contains 
+                 either the client's TLS/SSL X.509 certificate or the client's TLS/SSL 
+                 certificate and key.
+              
+             * - Client Key Password
+               - No 
+               - If the *Client Private Key* is protected with a password, 
+                 you must provide the password.
+
+
+          .. _azure-compass-encryption:
+
+          Azure
```

Comment:
As above.

Reply:
@mdb-ashley now it is actually ready for internal review!